src/lib/layers/editable-geojson-layer.js

```diff
@@ -247,9 +263,9 @@ export default class EditableGeoJsonLayer extends EditableLayer {
       return [];
     }
 
-    const layer = new GeoJsonLayer(
+    const layer = new this.props.GeometryLayer.Layer(
       this.getSubLayerProps({
-        id: 'draw',
+        id: `${this.props.GeometryLayer.id}-draw`,
```

Comment:
`getSubLayerProps` does this for you, so you can just leave it as `id: 'draw'`

Reply:
The reason I did this is to handle the case where someone is switching between layer definitions. Otherwise they'll have the same id in the deck stack right?

Reply:
The id only needs to be unique among the child layers of this composite layer. So, as long as `EditableGeoJsonLayer` only constructs one child layer with id `draw`, then it'll be unique.
https://github.com/uber/deck.gl/blob/af249ec08c029a49624458886f50be4fa7e9eeec/modules/core/src/lib/composite-layer.js#L91

Reply:
As far as I can tell, it won't update the sublayer if a new sublayer is defined with the same id. It wouldn't be necessary if you never toggle the sublayer, but in the example / in other feasible applications, you do.

https://github.com/uber/deck.gl/blob/master/modules/core/src/lib/layer-manager.js#L393